integration-tests/mondaynet.out

```diff
@@ -0,0 +1,8254 @@
+{
```

Comment:
not sure if this file should be commited

Reply:
fixed